charts/spire/charts/spire-agent/templates/configmap.yaml

```diff
@@ -20,6 +20,9 @@
 {{- end }}
 {{- define "spire-agent.yaml-config" -}}
 agent:
+  {{- if .Values.sockets.admin.enabled }}
+  admin_socket_dir: /tmp/spire-agent/private/admin.sock
```

Comment:
Private conveys the right security considerations, but not the reason one would use the socket.

Let's try to name things according to how they provide utility to the people using them, and not based on other characteristics that don't convey utility.  To clarify what that means, I don't go to a car dealership to buy "secure" I go to a car dealership to buy "transportation" that I could lock up (rendering it secure).  If I used this example in a naming pattern, "/tmp/spire-agent/transporation/car.sock" is preferred over "/tmp/spire-agent/secure/car.sock".

Please consider "/tmp/spire-agent/admin-api/admin.sock" or "/tmp/spire-agent/admin/admin.sock" or some other similar directory path instead of "/tmp/spire-agent/private/admin.sock"

Reply:
#176 
Istio should be using the workload api socket, aka the agent socket, so alias / naming of sockets for the purposes of istio compatibility seem out of place in considering the agent admin socket.

#177 
As far as I'm concerned, if one is attempting to run the spire-agent executable, they can pass on the command line the configuration file that will tell the executable where the config file exists, the same pattern we use in the Agent's primary Pod.  That config file will work even when the executable is being called as a client for a previously launched Agent service.

Neither of these items seem to have an impact on what we call the directory.

Reply:
I'm fine with the name as it stands then.  This was a call to improve upon the existing path, but there's value in matching the path to documentation elsewhere.